core/src/main/java/org/eclipse/hono/util/MessageDigestPasswordEncoder.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat Inc and others.
```

Comment:
Welcome to 2018 :stuck_out_tongue_winking_eye:

Reply:
Time flies 🙂